### DISCUSSION TITLE
Programming

### DISCUSSION THREAD

**Discussion by jdgp-hub** [community/community#125601] [Programming Help] [answered]
### Select Topic Area

Question

### Body

I can read HTML and other codes sort of. However, I do not know how to implement codes to my iPad. I’ve been collecting information on a hacker in my Wi-Fi who stole 500 notes from me which included mostly Dell information.

**Comment by arashtabaa** [comment, marked as answer]
### Step-by-Step Guide to Secure Your Wi-Fi

1. **Change Your Wi-Fi Password**:
   - Access your router's settings. This is typically done by entering the router's IP address (often `192.168.1.1` or `192.168.0.1`) into a web browser.
   - Log in using the router's admin credentials.
   - Navigate to the wireless settings section and change the Wi-Fi password to a strong, unique password.

2. **Update Router Firmware**:
   - In your router's settings, check for firmware updates. Keeping your router's firmware up to date helps protect against known vulnerabilities.

3. **Enable WPA3 or WPA2 Encryption**:
   - Ensure your Wi-Fi network is using WPA3 or WPA2 encryption (not WEP). This can usually be set in the security settings of your wireless configuration.

4. **Disable WPS (Wi-Fi Protected Setup)**:
   - WPS can be a security risk. Disable this feature in your router's settings.

5. **Monitor Connected Devices**:
   - Regularly check the list of connected devices in your router's admin panel. If you see any devices that you do not recognize, change your Wi-Fi password immediately.

6. **Use a Guest Network**:
   - Set up a separate guest network for visitors to prevent them from accessing your main network and its resources.

### Implementing and Running Code on Your iPad

To work with code on your iPad, you can use several applications and methods. Here are a few:

1. **Code Editor Apps**:
   - **Textastic**: A powerful code editor for iPad that supports syntax highlighting for many languages.
   - **Koder**: Another robust code editor that supports a wide range of languages and offers features like SFTP/FTP for web development.
   - **Pythonista**: If you're working with Python, this app provides a comprehensive development environment.

2. **Using a Cloud Service**:
   - **GitHub**: Store your code in repositories on GitHub. You can use the GitHub app to view and edit your code.
   - **Replit**: An online IDE that allows you to write and run code directly in your browser. You can use it on your iPad.

3. **Connecting to a Remote Server**:
   - Use an SSH client like **Blink Shell** or **Termius** to connect to a remote server where you can develop and run your code.

### Example: Setting Up Textastic for Basic HTML Coding

1. **Download and Install Textastic**:
   - Go to the App Store on your iPad and download Textastic.

2. **Create a New HTML File**:
   - Open Textastic, tap the "+" button to create a new file.
   - Select "New File" and choose "HTML" from the list of supported file types.
   - Enter your HTML code in the editor.

3. **Preview Your HTML File**:
   - Tap the "Preview" button to see how your HTML code renders in a browser.

Here is an example of a simple HTML file you can create:


```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Sample Page</title>
</head>
<body>
    <h1>Welcome to My Page</h1>
    <p>This is a simple HTML example.</p>
</body>
</html>
```

- By following these steps, you can secure your Wi-Fi network and start working with code on your iPad using the recommended applications and methods.

-  Install and configure a code editor app like Textastic on your iPad.
-  Set up a GitHub account to store and manage your code projects.

**Reply by jdgp-hub** [reply]
Thank you

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @jdgp-hub ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
